spec/mongo/cluster_spec.rb

```diff
@@ -20,6 +20,24 @@
 
   let(:cluster) { cluster_without_io }
 
+  describe 'initialize' do
+
+    context 'when there are duplicate addresses' do
+
+      let(:addresses) do
+        SpecConfig.instance.addresses + SpecConfig.instance.addresses
+      end
+      let(:cluster_with_dup_addresses) do
+        described_class.new(addresses, monitoring, SpecConfig.instance.test_options)
+      end
+
+      it 'does not error out' do
+        cluster_with_dup_addresses
+      end
```

Comment:
Would you mind making this test more specific by:
1. Changing the description to `it 'does not raise an exception'`
2. Updating the test body to specifically test that the code doesn't raise an error
```
expect { cluster_with_dup_address }.not_to raise_error
```

Reply:
Thanks, I made these.